ctapipe_io_lst/_version.py

```diff
@@ -0,0 +1,5 @@
+# coding: utf-8
+# file generated by setuptools_scm
+# don't change, don't track in version control
+version = '0.6.0'
```

Comment:
why is this version `0.6.0`?

Reply:
Dam, accidental commit. File should not have been added.